.travis.yml

```diff
@@ -8,19 +8,18 @@ cache:
 install:
   - pip3 install -r requirements.txt
```

Comment:
In MLCAT, we are not installing any OS-level packages like graphviz. Are they not needed at all for functional tests?

Reply:
The functional tests seem to run without them, maybe not required